docs/0_concepts.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Serverless Concepts
+description: todo
+layout: Page
+---
+
+# Concepts
+
+Before we begin, let's run through some of the main concepts behind serverless.
+
+- [Serverless Functions](#serverless-functions)
+- [Serverless Events](#serverless-events)
+- [Serverless Resources](#serverless-resources)
+- [Serverless Services](#serverless-services)
+- [Serverless Plugins](#serverless-plugins)
+
+# Serverless Functions
+
+Functions are the unit of deployment in a serverless application
```

Comment:
Right now services are the units of deployment in the Serverless framework. Don't know if the architecture or the framework is meant here. (Additionally there's a `. ` missing at the end of the sentence).